lib/worker/index.js

```diff
@@ -29,10 +36,17 @@ Object.keys(queues).forEach(name => {
 
   queue.on('active', function (job, jobPromise) {
     app.log.info(`Job started name=${name} id=${job.id}`)
+    job.meta_time_start = new Date()
   })
 
   queue.on('completed', function (job, jobPromise) {
     app.log.info(`Job completed name=${name} id=${job.id}`)
+    measureElapsedTime(job.meta_time_start, { queue: name, status: 'completed' })
+  })
+
+  queue.on('failed', async (job, err) => {
+    app.log.error(`Error occurred while processing job id=${job.id} on queue name=${name}`)
+    measureElapsedTime(job.meta_time_start, { queue: name, status: 'failed' })
```

Comment:
Should we `measureElapsedTime` for the `error` as well?

Reply:
From what I read, `error` is thrown when bull has trouble reading from the queue. So I avoided it in that case. There isn't a job available to read the `startTime` from.

Reply:
Makes perfect sense!